viewer/multies.js

```diff
@@ -875,26 +892,30 @@ nodes.forEach((node) => {
   if (node.toLowerCase().includes(',http')) {
     console.log('WARNING - multiESNodes may be using a comma as a host delimiter, change to semicolon');
   }
-  clients[node] = new ESC.Client({
-    host: node.split(',')[0],
-    apiVersion: '7.7',
+
+  let nodeName = node.split(',')[0];
+
+  // TODO - 127.0.0.1:9200 throws error: "TypeError [ERR_INVALID_URL]: Invalid URL: 127.0.0.1:9200"
+  nodeName = nodeName === '127.0.0.1:9200' ? 'http://localhost:9200' : nodeName;
```

Comment:
@awick I need your wisdom here!?!?!

Reply:
If it doesn't start with "http" add http://

Reply:
Ahhhh 👍